deploy/terraform/main.tf

```diff
@@ -181,7 +181,7 @@ resource "aws_ecs_service" "gp2gp_ps" {
   network_configuration {
     subnets = [for subnet in aws_subnet.nia_gp2gp_public_subnet : subnet.id]
     assign_public_ip = "true"
-    security_groups = [aws_security_group.nia_gp2gp_dmz.id]
+    security_groups = [aws_security_group.nia_gp2gp_dmz.id, aws_security_group.ps_db_migration.id]
```

Comment:
Was this necessary?

Reply:
yes, as otherwise ECS tasks don't have an access to DB or I had to give full access to DB manually. Not sure how it worked before. In general, we need to look a bit closer at networking and security and reorganize it a bit I think.

Reply:
I vaguely remember talking to Danny about some DB networking, my view being that connectivity wouldn't work with the design, but it seemed to work when tested.  So it makes sense that our testing was at fault.